src/components/Text/index.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import styled from 'styled-components';
+import { HTMLAttributes } from 'react';
+
+type HeadingAsTagProps = {
+  asTag?: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
+};
+
+type TextProps = {
+  size?: 'xxl' | 'xl' | 'l' | 'm' | 's' | 'xs' | 'xxs';
+} & HTMLAttributes<HTMLHeadingElement>;
+
+const StyledHeading = styled.h1<TextProps>`
+  font-weight: 600;
```

Comment:
Do they all have this font-weight?

Reply:
For now they do, but as mentioned, we can update this in the future.